src/main/java/org/mule/extension/http/api/listener/server/RequestAddressesFormatValueProvider.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.http.api.listener.server;
+
+import static com.google.common.collect.Sets.newHashSet;
+import static org.mule.runtime.extension.api.values.ValueBuilder.newValue;
+
+import org.mule.runtime.api.value.Value;
+import org.mule.runtime.extension.api.values.ValueBuilder;
+import org.mule.runtime.extension.api.values.ValueProvider;
+
+import java.util.Set;
+
+public class RequestAddressesFormatValueProvider implements ValueProvider {
```

Comment:
What's the benefit of using value providers here?

Reply:
It shows nicer values in the select in Studio